frontend/src/api/Language.ts

```diff
@@ -0,0 +1,38 @@
+enum Language {
+  Python = 'PYTHON',
+  Ruby = 'RUBY',
+  Swift = 'SWIFT',
+  CPP = 'CPP',
+  PHP = 'PHP',
+  C = 'C',
+  Java = 'JAVA',
+  JavaScript = 'JAVASCRIPT',
+  Rust = 'RUST',
+  Bash = 'BASH',
+}
+
+export const fromString = (key: string): Language => Language[key as keyof typeof Language];
+
+export const languageToEditorLanguage = (key: Language): string => {
+  // Swift, Rust, and Bash do not have corresponding editor languages.
+  switch (key) {
+    case Language.Python:
```

Comment:
nit: Comment out the languages that we aren't currently supporting

Reply:
Good idea -- done!